### ISSUE TITLE
How to use properly the query object for multiple ids?

### ISSUE THREAD

**Issue by giggioz** [raix/push#43]
Hi,

i'm struggling with the query object, i want to use it to send a push to multiple ids of Meteor.users collection.

Let's say i have an array ```users = [ 'id1', 'id2', 'id2'];```

I tried to set the push.send like this

```
//in my method pushAddExpense
Push.send({
				from: 'x',//not used
				title: 'Expense Added',
				text: 'title',
				badge: 1,
				payload: {
					groupId: groupId,
					userId: userId
				},
				query: {
					userId: {$in : users} 
				}
			});
```
As you can see i'm using mongo syntax to make the query object but i get the following error

```Exception while invoking method 'pushAddExpense' Error: key $in must not start with '$' ```

In the stack trace 
```
at [object Object].Mongo.Collection.(anonymous function) [as insert] (packages/mongo/collection.js:574:1)
```
Probably i'm missing something, am i?

**Comment by raix** [comment]
duplicate of #39

**Comment by raix** [comment]
Hi @giggioz I've made a fix for this issue, can you give `raix:push@2.6.3-pre.1` a go to confirm if it's solving the issue?

**Comment by dennisharrison** [comment]
Works here.

**Comment by giggioz** [comment]
Hi, i still have the problem, here's my settings : 

NOTE: i'm using Meteor 1.0.4.1 now
Using raix:push@2.6.3

My Method 

```
Push.send({
				from: 'x',//not used
				title: 'Expense Added',
				text: username + ' added an expense in ' + groupName,
				badge: 1,
				payload: {
					groupId: groupId,
					userId: userId
				},
				query: {
					userId: {$in : users}
				}
			});
````

my server log

**SENDING PUSH users = [ 'e7u7AdRdFjKRc8NLc', 'wK666qLdSwyBE3zQz', 'fXMMpMXEJnsLJ6wLr' ]**

Exception while invoking method 'pushAddExpense' Error: Match error: Expected object, got string in field query
at checkSubtree (packages/check/match.js:275:1)
 at packages/check/match.js:296:1
at Function._.each._.forEach (packages/underscore/underscore.js:113:1)
at checkSubtree (packages/check/match.js:290:1)
at check (packages/check/match.js:32:1)
at _validateDocument (packages/raix:push/lib/common/notifications.js:9:1)
at EventEmitter.Push.send (packages/raix:push/lib/common/notifications.js:67:1)
at [object Object].Meteor.methods.pushAddExpense (app/lib/methods.js:470:9)
at [object Object].methodMap.(anonymous function) (packages/meteorhacks:kadira/lib/hijack/wrap_session.js:160:1)
at maybeAuditArgumentChecks (packages/ddp/livedata_server.js:1617:1)

**Comment by raix** [comment]
Thanks @giggioz Should be fixed now in 2.6.4
